src/main/java/com/seals/camplanner/event/controllers/EventController.java

```diff
@@ -37,10 +47,18 @@ protected Location convert(Long locationId) {
         return toEventDtoList(eventService.findAll());
     }
 
-    //TODO add treatment for the "location" parameter
     @PostMapping("/event")
-    public EventDto saveEvent(@RequestBody EventDto event) {
```

Comment:
I saw that you changed the response status for the delete request, should we change this one to `201 CREATED` as well?

Reply:
Yeap. Forgot this one.